simulator-core/src/test/java/com/evbox/everon/ocpp/it/AuthenticationIt.java

```diff
@@ -0,0 +1,112 @@
+package com.evbox.everon.ocpp.it;
```

Comment:
the pattern for all our integrations tests are 
package name: ocpp header
class name: ocpp use case


so please move this to security/UpdateCharginStation...

Reply:
done